.github/workflows/run-core-traits-tests.yml

```diff
@@ -23,9 +23,9 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install local package
       run: |
-        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install --upgrade pip
         python -m pip install .
-        python -m pip uninstall -y setuptools wheel
+        python -m pip list
```

Comment:
I'll remove this line once I've confirmed its results. It's not clear to me whether the GitHub Actions Python environment includes `setuptools` or not. If it does, we may still want to explicitly uninstall it so that we catch any accidental dependencies of the Traits source on `setuptools`.

Reply:
Actually, let's keep the `python -m pip list` - it may be useful for future debugging.